### ISSUE TITLE
Apply Profile header to the Profile folder - not Our People folder

### ISSUE THREAD

**Issue by sharrocksb** [adobe/design-website#88]
There are long form articles in the 'Our People' folder that should have the standard header. Is it possible to apply the style only to the ['Profile' folder](https://adobe.sharepoint.com/:f:/r/sites/Adobe.design/Shared%20Documents/website/stories/our-people/profile?csf=1&web=1&e=7nu3On) within the 'Our People' folder. 

Example: Khoi's article should have a standard header not the profile header: https://main--design-website--adobe.hlx3.page/stories/our-people/when-designers-give-each-other-books

Laura Herman's article is the other article

NOTE: Patrick - let me know how time intensive this issue is and I can reprioritize as needed.

**Comment by pfulton** [comment]
Yes, I think this is possible to do, and will probably take a few hours to accomplish. Possibly fewer if I can reuse existing styles (which I think I can)
